### PR TITLE
Fixed  Style/UnneededInterpolation

### LINKED ISSUE #1823: Fix Style/UnneededInterpolation offenses
Fix Style/UnneededInterpolation Rubocop cop offenses and remove the exclusion from the todo file. :bowtie: 

This cop checks for strings that are just an interpolated expression. It support autocorrection.

https://rubocop.readthedocs.io/en/latest/cops_style/#styleunneededinterpolation

### PR DESCRIPTION
**Pull request by SidharthBansal**
Fixes #1823
Changed unneeded interpolations and  removed exclusions from the todo file

### CODE REVIEW

File: spec/controllers/admin/conferences_controller_spec.rb [hunk 1]
```diff
@@ -56,8 +56,8 @@
           conference.reload
           expect(flash[:error])
               .to eq("Updating conference failed. Short title can't be blank.")
-          expect(conference.title).to eq("#{conference.title}")
-          expect(conference.short_title).to eq("#{conference.short_title}")
+          expect(conference.title).to eq(conference.title.to_s)
```
Comment (Ana06) [inline spec/controllers/admin/conferences_controller_spec.rb]: `conference.title` is already a string so `to_s` is not needed 😉

File: app/helpers/format_helper.rb [hunk 2]
```diff
@@ -122,7 +122,7 @@ def word_pluralize(count, singular, plural = nil)
              plural || singular.pluralize
            end
 
-    "#{word}"
+    word.to_s
```
Comment (Ana06) [inline app/helpers/format_helper.rb]: word is always a string, `to_s` is not needed.

**Comment by SidharthBansal** [comment]
Thanks for the suggestions. I will keep in mind for next time. I will do the required changes.

**Comment by SidharthBansal** [comment]
There has been an error with my fork. I'll open a new PR for this issue soon. 
Thanks

**Pull request closed without merging**